src/components/ebay-menu/examples/01-basic/template.marko

```diff
@@ -1,4 +1,9 @@
 <ebay-menu text="eBay Menu">
+    <ebay-menu-label>
+        <span class="flag" style="background-position: 0 -25px">
```

Comment:
Why is the background-position style needed? 
I would recommend in examples to try to stray away from using tags and such that generally aren't recommended to use (I know style is one you generally should stray away from using).

Reply:
We should leave the basic example untouched, for backwards compatibility. A new example should be created showing how to use this new feature.